x-pack/plugins/maps/public/classes/styles/vector/properties/style_property.ts

```diff
@@ -24,6 +24,7 @@ export interface IStyleProperty {
   getStyleName(): VECTOR_STYLES;
   getOptions(): StylePropertyOptions;
   renderRangeLegendHeader(): ReactElement<any> | null;
+  renderBreakedLegend(): ReactElement<any> | null;
```

Comment:
Remove from interface. `renderLegendDetailRow` should pass style props to component so this method is not needed.